cli-utils/self-managed-docs-branch.js

```
@@ -0,0 +1,84 @@
+  const url = 'https://raw.githubusercontent.com/redpanda-data/docs/main/antora.yml'
+  return new Promise((resolve, reject) => {
+    https.get(url, res => {
+      if (res.statusCode !== 200) {
+        return reject(new Error(`Failed to fetch antora.yml: ${res.statusCode}`))
+      }
+      let body = ''
+      res.on('data', chunk => (body += chunk))
+      res.on('end', () => {
+        try {
+          const cfg = yaml.load(body)
+          if (typeof cfg.version == null) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)